mypy.sh

```
@@ -8,25 +8,31 @@ RETVAL=0
 
 echo "running $SCRIPT_DIR/mypy.sh"
 
+export MYPYPATH=${SCRIPT_DIR}/stubs
+MYPY="mypy --strict --no-incremental --show-error-codes --pretty"
+
+echo cd $SCRIPT_DIR/engine
 cd $SCRIPT_DIR/engine
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)